kabob-build/src/main/resources/edu/ucdenver/ccp/kabob/build/rules/temp/drugbank/drugbank_rules.clj

```diff
@@ -1,99 +1,101 @@
+         ;; create a subclass of the target that will participate in this
+         ;; particular binding
+         (?/target rdfs/subClassOf ?/target_bioentity)
 
-`{:name "drugbank-gene-drug-relation"
-  ;; this need to be hashed and have proper classes
-  :head (;(?/gorgporv kiao/STANDIN_gene_drug ?/drug);)
-         ;(?/gorgporv kiao/STANDIN_drugbank_gene_drug ?/drug);)
-         (?/interaction rdfs/subClassOf obo/MI_0000) ;interaction
+          ;; link the binding event to the target participant (protein in this case)
+         (?/r_target_participant rdf/type owl/Restriction)
+         (?/r_target_participant owl/onProperty obo/RO_0000057) ; has_participant
+         (?/r_target_participant owl/someValuesFrom ?/target)
 
-         ;;these two triples are redundant rdf macros
-         ;(?/interaction kbio/rsv_has_participant ?/gorgporv)
-         ;(?/interaction kbio/rsv_has_participant ?/drug)
-         
-         (?/r1 rdf/type owl/Restriction)
-         (?/r1 owl/onProperty obo/RO_0000057) ; has_participant
-         (?/r1 owl/someValuesFrom ?/drug)
+         ;; create a subclass of the drug molecule that will participate
+         ;; in this particular binding
+         (?/drug rdfs/subClassOf ?/drug_bioentity)
 
-         (?/r2 rdf/type owl/Restriction)
-         (?/r2 owl/onProperty obo/RO_0000057) ; has_participant
-         (?/r2 owl/someValuesFrom ?/gorgporv)
+          ;; link the binding event to the drug
+         (?/r_drug_participant rdf/type owl/Restriction)
+         (?/r_drug_participant owl/onProperty obo/RO_0000057) ; has_participant
+         (?/r_drug_participant owl/someValuesFrom ?/drug)
 
-         (?/r3 rdf/type owl/Restriction)
-         (?/r3 owl/onProperty obo/BFO_0000055) ; realizes
-         (?/r3 owl/someValuesFrom ?/inheres)
+          ;; this binding realizes the drug_role that inheres_in the drug
+         (?/r_realizes_drug_role rdf/type owl/Restriction)
+         (?/r_realizes_drug_role owl/onProperty obo/BFO_0000055) ; realizes
+         (?/r_realizes_drug_role owl/someValuesFrom ?/inheres)
 
          ;;create a new anonymous class that is CHEBI drug role
-         ;;   that inheres in this specific drug
+         ;;   that inheres_in in this specific drug
          ;;   so that it can be realized by the restriction above
-         (?/r4 rdf/type owl/Restriction)
-         (?/r4 owl/onProperty obo/RO_0000052) ;inheres_in
-         (?/r4 owl/someValuesFrom ?/drug)
-         
-         (?/inheres rdfs/subClassOf obo/CHEBI_23888) ;drug role
-         (?/inheres rdfs/subClassOf ?/r4)
+         (?/r_inheres rdf/type owl/Restriction)
+         (?/r_inheres owl/onProperty obo/RO_0000052) ;inheres_in
+         (?/r_inheres owl/someValuesFrom ?/drug)
+
+         (?/inheres rdfs/subClassOf obo/CHEBI_23888) ;drug (role)
+         (?/inheres rdfs/subClassOf ?/r_inheres)
 
          ;;make the interaction be necessarily part of these 3 restrictions
          ;;still need to add a sufficient definition
```

Comment:
Do you have any idea what this comment means?  What's might still be missing from this definition?